app/ContactListItem.js

```diff
@@ -0,0 +1,89 @@
+'use client';
+import React from 'react'
+import { useContext } from 'react';
+import UserContext from './contexts/UserContext';
+import { useRouter } from 'next/navigation';
+import ImageWithDefault from './ImageWithDefault';
+
+const ContactListItem = (props) => {
+
+  const context = useContext(UserContext);
+  const router = useRouter();
+
+  const handleRowClick = (id, e) => {
```

Comment:
I would recommend not using abbreviations on naming conventions just so its easier to understand